frappe/public/js/frappe/form/controls/int.js

```diff
@@ -2,21 +2,14 @@ frappe.ui.form.ControlInt = class ControlInt extends frappe.ui.form.ControlData
 	static trigger_change_on_input_event = false;
 	make() {
 		super.make();
-		// $(this.label_area).addClass('pull-right');
-		// $(this.disp_area).addClass('text-right');
 	}
 	make_input() {
 		var me = this;
 		super.make_input();
-		this.$input
-			// .addClass("text-right")
-			.on("focus", function () {
-				setTimeout(function () {
-					if (!document.activeElement) return;
-					document.activeElement.select();
-				}, 100);
-				return false;
-			});
+		this.$input.on("focus", function () {
+			document.activeElement?.select?.();
```

Comment:
Also shouldn't this just be $input.select() ? Why are we relying on active element in DOM when we specifically know what user focused on?

Reply:
Yeah I don't really why it's there in the first place, and indeed `this.input.select()` should be equivalent.